src/pages/chat/chat.ts

```diff
@@ -5,40 +5,54 @@ import { SelectChat } from "./selectChat";
 import { Input } from "../../components/input";
 import {Btn} from "../../components/btn";
 import "./chat.scss";
-import { Block } from "../../utils";
-import router from "../../router";
+import {Block} from "../../utils/block";
+import router from "../../router/Router";
 import { store } from "../../store";
 import { LoginController, ChatController, IChatData } from "../../controllers";
 import { Modal } from "../../components/modal"
 import { Form } from "../../components/form";
 import chatItemTemplate from "./chatItem.tmpl";
-import { avatarIconBase64 } from "../../utils";
+import { avatarIconBase64 } from "../../utils/constants";
 
 const loginController = new LoginController();
 const chatController = new ChatController();
 
 export const showModal = async (modalId: string) => {
-    const modal = document.querySelector(`.modal[data-id = "${modalId}"]`);
-    const popover = document.querySelector(`.chat-open__popover[data-id = "${modalId}"]`);
-    const overlay = document.querySelector(".overlay");
-    if (modal?.classList.contains("hidden")) {
-        modal?.classList.remove("hidden");
-        overlay?.classList.remove("hidden");
+    const modal = document.querySelector(`.modal[data-id = "${modalId}"]`) as HTMLElement;
+    const popover = document.querySelector(`.chat-open__popover[data-id = "${modalId}"]`) as HTMLElement;
```

Comment:
Нужно стараться избегать явного приведения типов там, где этого избежать можно. В данном случае можно написать так:
```suggestion
    const popover = document.querySelector<HTMLElement>(`.chat-open__popover[data-id = "${modalId}"]`);
```